crates/bevy_winit/src/lib.rs

```diff
@@ -263,9 +264,52 @@ pub fn winit_runner_with(mut app: App) {
                 window_id: winit_window_id,
                 ..
             } => {
-                let world = app.world.cell();
-                let winit_windows = world.get_resource_mut::<WinitWindows>().unwrap();
-                let mut windows = world.get_resource_mut::<Windows>().unwrap();
+                let mut system_state: SystemState<(
```

Comment:
I think i agree with clippy on this one, this type is horrendeous. I'd recommend making a custom struct and using `#[derive(SystemParam)]` so it's neater.